packages/view/backend/sirius-components-view-diagram/src/main/resources/model/diagram.ecore

```diff
@@ -203,7 +203,7 @@
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="labelIcon" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="NodeStyleDescription" abstract="true"
-      interface="true" eSuperTypes="#//Style #//BorderStyle"/>
+      interface="true" eSuperTypes="#//BorderStyle"/>
```

Comment:
Instead of just changing the name of the feature, you have also changed the structure of the meta model, why?

Reply:
Ok, I'm not sure it really made much sense to have a common style for nodes, edges and labels. It's not as if we were manipulating such an abstraction frequently. I'm not even sure that BorderStyle is really relevant.